docs/index.md

```diff
@@ -174,7 +174,7 @@ do
     deployment_date=$(git log --format="%at" ${deployment_sha} | head -n 1)
 
     # Skip deployments before date
-    if [ ${deployment_date} -lt $(date -d "Oct 01 2020" +%s) ]
+    if [ ${deployment_date} -lt $(date -jf "%b %d %Y" "Oct 01 2020" +%s) ]
```

Comment:
This is not valid in linux which I think will be the majority of usages.

Reply:
Ok. I'll add it as a comment then.